bip-emilengler-redefinition-of-the-term-address.mediawiki

```diff
@@ -0,0 +1,54 @@
+<pre>
+  BIP: UNDEFINED
+  Titel: Redefinition of the term address
+  Author: Emil Engler <me@emilengler.com>
+  Status: Draft
+  Type: Informational
+  Created: 2019-10-17
+  License: Public Domain
+</pre>
+
+==Abstract==
+This BIP proposes a new term for 'address'
+
+==Specification==
+The new term is:
+''Bitcoin'' '''Invoice''' ''Address''
+Short: ''BIA''
```

Comment:
I think BIA causes more confusion than it serves as a easily recognizable identifier. Also it will be misleading, when in the (far) future, it will just be "Bitcoin invoice".

Reply:
You might be right. I think invoice is short enough